backend/src/main/java/com/bakdata/conquery/sql/conversion/model/SelectsIds.java

```diff
@@ -0,0 +1,94 @@
+package com.bakdata.conquery.sql.conversion.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import javax.annotation.Nullable;
+
+import com.bakdata.conquery.sql.conversion.SharedAliases;
+import lombok.Getter;
+import lombok.NonNull;
+import org.jooq.Condition;
+import org.jooq.Field;
+import org.jooq.impl.DSL;
+
+public class SelectsIds implements Qualifiable<SelectsIds> {
```

Comment:
der Name ist etwas merkwürdig, sit das ein Typo?

Reply:
Meine Herleitung war: das sind die IDs für die `Selects` Klasse, also hab ich `SelectsIds` draus gemacht.
Du meinst Typo im Sinne dass es `SelectIds` heißen sollte?
Ansonsten würde ich anbieten: 
- `SqlIds`
- `SqlIdColumns`

Reply:
@awildturtok Mir auch, hab die Klasse umbenannt. Ist sonst noch was offen?

Reply:
die Tests hast du aus den legacy tests kopiert oder? Wenn ja dann passt. Sonst würde ich nochmal drüber gehen.

Reply:
genau, sind 1 zu 1 übernommen, außer dass ich `DATE_RANGE` validity dates in start und end DATE column umgewandelt hab, sodass es für beide Dialekte passt.